analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ExecutingSqlQueries_Net46.cs

```diff
@@ -106,7 +110,7 @@ public void OdbcCommands(OdbcConnection connection, OdbcTransaction transaction,
             OdbcDataAdapter adapter;
             adapter = new OdbcDataAdapter(); // Compliant
             adapter = new OdbcDataAdapter(command); // Compliant
-            adapter = new OdbcDataAdapter(query, ""); // Compliant
+            adapter = new OdbcDataAdapter(query, $" non constant {query}"); // Compliant
```

Comment:
This is little confusing, as it looks that the type has two query parameters and it's not clear why we don't raise on the second one.
```suggestion
            adapter = new OdbcDataAdapter(query, $"concatenated connection string {query}"); // Compliant
```
or with new parameter in the method
```suggestion
            adapter = new OdbcDataAdapter(query, $"connection string {server}"); // Compliant
```

Reply:
Applies to all